Re-import css/mediaqueries WPT

CYBAI opened the pull request:
#### 884b2e26cb219b9dac284e83bd4b5372875da7d4
<pre>
Re-import css/mediaqueries WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=258438">https://bugs.webkit.org/show_bug.cgi?id=258438</a>

Reviewed by Tim Nguyen.

Import based on dc23a29 commit in WPT.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/device-aspect-ratio-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/device-aspect-ratio-005-expected.xht: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/device-aspect-ratio-005.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/dynamic-range-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/dynamic-range.html:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/inverted-colors-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/inverted-colors.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/match-media-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/match-media-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-resolution-expected.xht: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/device-aspect-ratio-001-expected.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-resolution.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-non-matching-lazy-load.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-non-matching-lazy-load.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/overflow-media-features-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/overflow-media-features.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-reduced-transparency-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-reduced-transparency.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/resources/mq-non-matching-lazy-load-style.css: Added.
(body):
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/update-media-feature-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/update-media-feature.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/265647@main">https://commits.webkit.org/265647@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b70e78ca0b6b1e4e028dc8b973c5a038c8c9af59

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/11202 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/11412 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/11743 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/12853 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/10682 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/13791 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/11397 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/13597 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/11363 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/12248 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/9462 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/13258 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/9542 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/10138 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/17338 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/10612 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/10293 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/13510 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/10723 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/8810 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/9893 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/10029 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/2761 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/14167 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/10576 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

CYBAI commented on LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-non-matching-lazy-load.tentative-expected.txt:
```diff
@@ -0,0 +1,3 @@
+
+FAIL Ensure browser renders the page without waiting for non matching @import style assert_less_than: expected a number less than 1000 but got 1011
```
hmm, this test seems to generate different results on my machine (macOS 13.4) every time. I'm wondering even it could result in flaky tests even if it passes on CI. As it's still `tentative`, should I remove it for now and maybe sync it when it's non-tentative? or, should I add it to the skip list in `TestExpectations`?

nt1m replied:
I would just skip the test due to unstable output.

CYBAI replied:
Added it to the skip list 🙏 

https://github.com/WebKit/WebKit/pull/15233/files#diff-0b438e462819a7b0d06250189152c7bcfb07b7c39acd73938bdfccc7b25b8058R1662-R1663

CYBAI committed:
Re-import css/mediaqueries WPT
https://bugs.webkit.org/show_bug.cgi?id=258438

Reviewed by Tim Nguyen.

Import based on dc23a29 commit in WPT.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/device-aspect-ratio-001.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/device-aspect-ratio-005-expected.xht: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/device-aspect-ratio-005.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/dynamic-range-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/dynamic-range.html:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/inverted-colors-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/inverted-colors.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/match-media-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/match-media-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-resolution-expected.xht: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/device-aspect-ratio-001-expected.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-calc-resolution.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-non-matching-lazy-load.tentative-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/mq-non-matching-lazy-load.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/overflow-media-features-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/overflow-media-features.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-reduced-transparency-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/prefers-reduced-transparency.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/resources/mq-non-matching-lazy-load-style.css: Added.
(body):
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/update-media-feature-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/update-media-feature.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/mediaqueries/w3c-import.log:

Canonical link: https://commits.webkit.org/265647@main
